empyrical/stats.py

```diff
@@ -140,11 +142,11 @@ def cum_returns(returns, starting_value=0):
     if len(returns) < 1:
         return type(returns)([])
 
-    if np.isnan(np.asanyarray(returns)[0]):
+    if np.any(np.isnan(np.asanyarray(returns)[0])):
```

Comment:
We should only catch the case where the whole first row are `nan`s, as it's the only case produced by `.pct_change()`.